app/auth/authenticate.js

```diff
@@ -5,7 +5,7 @@ import logger from '../utils/logger.js'
 
 export async function getJwtPublicKey (kid) {
   const client = jwksClient({
-    jwksUri: `https://login.microsoftonline.com/${process.env.API_TENANT_ID}/discovery/v2.0/keys`
+    jwksUri: `https://login.microsoftonline.com/${process.env.RP_INTERNAL_APIM_TENANT_ID}/discovery/v2.0/keys`
```

Comment:
This is not right, this is authentication for our API and should not change

Reply:
My mistake, global search and replace. think ive sorted it now....